internal/collector/deployment.go

```diff
@@ -227,6 +229,27 @@ var (
 				}
 			}),
 		},
+		{
+			Name: "kube_deployment_revision",
+			Type: metric.Gauge,
+			Help: "Sequence number representing a specific revision of the deployment controller represented by the annotaion 'deployment.kubernetes.io/revision'.",
```

Comment:
```suggestion
			Help: "Sequence number representing a specific revision of the deployment controller represented by the annotation 'deployment.kubernetes.io/revision'.",
```

Reply:
Fixed typo and changed from Sequence number into Number, because the revision is not really a sequence.